SPECS/libvirt/libvirt.spec

```diff
@@ -121,36 +125,50 @@ make check
 %{_datadir}/libvirt/api/libvirt-lxc-api.xml
 
 %files docs
-/usr/share/augeas/lenses/*
-/usr/share/libvirt/*
-/usr/share/doc/libvirt/*
-/usr/share/locale/*
+%{_datadir}/augeas/lenses/*
+%{_datadir}/libvirt/*
+%{_docdir}/libvirt/*
+%{_datadir}/locale/*
 %{_mandir}/*
 
 %changelog
+* Tue Jun 15 2021 Nicolas Ontiveros <niontive@microsoft.com> - 6.1.0-3
+- Patch CVE-2020-10701
```

Comment:
nit: Maintain indentation consistence on the changelog. Maybe mention the spec linting too.